### PR TITLE
Ms/ml dev 1.2

### PR DESCRIPTION
**Pull request by mstefferson**
1.2 release. This is built off of ms/reorg_dirs. We have switch to the requirements format, and I have included packages discussed by me and SD, as well as some from previous client work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+
+The various docker images are tagged by their repository name under their parent directory. E.g., The Dockerfile for docker-ml-dev:1.0 corresponds to docker-ml-dev/1.0/Dockerfile
+
+### docker-ml-dev
+
+This is currently the base development image used in conjunction with [docker-cookiecutter-datascience](https://github.com/manifoldai/docker-cookiecutter-data-science).
+
+### orbyter-base-sys
+
+Base docker image for ML development in python. ML images are build on top of this, and this should not be used directly for development.
```
Comment (resdntalien) [inline README.md]: Ok this is clearer than the other PR. I would still want one more sentence to explain the design decision to factor the images into these two pieces.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
+
+### orbyter-dl-dev
+
+Docker image for DL development. 
+
+## Release info
+
+The directories in the top level contain the different images. Within each directory are subdirectories of version releases, which should match [Manifold's Dockerhub](https://hub.docker.com/u/manifoldai/). For example, the Dockerfile for manifoldai/orbyter-ml-dev:1.2 is orbyter-ml-dev/1.2/Dockerfile. 
+
+When build a new release, build a new directory. Update a requirements file only with top level modules.
```
Comment (resdntalien) [inline README.md]: Can you make this another section? 
Also can you add the Docker commands to actually "build" the image here? I always forget how to do it and have to look it up.

Comment (mstefferson) [reply README.md]: Updated in new push

File: README.md (hunk 1 above)
Comment (mstefferson) [reply README.md]: Updated in new push

**Pull request merged**